### ISSUE TITLE
thetimes.co.uk no longer working

### ISSUE THREAD

**Issue by RainmakerRaw** [iamadamdev/bypass-paywalls-chrome#1041]
###
**1. First search [Issues](https://github.com/iamadamdev/bypass-paywalls-chrome/issues) to see if your problem has already been reported.**
Previous issues by other people closed, citing unable to replicate. Last one in August 2020.

**2. Make sure uBlock Origin is installed.**
v1.31.0 in hard mode. The Times is set to allow JS, with 3p noop and 3p-scripts noop.

**3. Chrome or Firefox?**
Both -
Firefox v83, and Nightly 85.0a1 (2020-11-29) (64-bit)
Ungoogled Chromium 84.0.4147.135

**4. Paste the article URL for the site you are having issues with.**
[The Times](https://www.thetimes.co.uk/edition/world/amazon-paintings-reveal-ice-age-civilisation-lvzb8xtb0).

**5. Describe the issue.**
Running Bypass Paywalls v1.7.8 from Releases. Underlying OS is Ubuntu 20.04.1.

One would expect the main body of paywalled articles to load and be readable (as they have in the past). Now, the paywall remains in place blocking the main article body, requesting a subscription to continue reading. See this [IMAGE](https://i.ibb.co/kmT7cq0/Screenshot-20201201-121336.png) for clarification.

Firefox is run with `privacy.resistFingerprinting` enabled, but this shouldn't be a factor as the issue is also present in Chromium.
###

**Comment by iamadamdev** [comment]
That article works for me on Chrome and FF (after agreeing to the cookie popup).

**Comment by RainmakerRaw** [comment]
> That article works for me on Chrome and FF (after agreeing to the cookie popup).

Strange. I don't know what else to try. I've tried a clean profile, disabled all addons except Bypass Paywalls and uBO (which I also tried disabled), tried different DNS, three different browsers, with VPN and without... Several other people report the same issue with The times so there's clearly a variable at play that I've not found yet. I'll keep experimenting.
